load-test/gtfs_user_test.py

```diff
@@ -0,0 +1,41 @@
+from locust import HttpUser, TaskSet, task, between
```

Comment:
Great addition! Could you provide some documentation on how to setup/run the load tests locally?
Suggestion: Create or add to an existing GH action the load tests using the cli